include/dg/llvm/analysis/ValueRelations/GraphElements.hpp

```diff
@@ -29,16 +29,20 @@ class VROp {
     virtual ~VROp() = default;
 
 #ifndef NDEBUG
-    virtual void dump() const = 0;
+    virtual std::string toStr() const = 0;
```

Comment:
What is the advantage of having this `toStr` method?

Reply:
Part of it is probably to avoid a tiny bit of code duplication (can dump in generalDump). I also feel that I there was another reason but right now, I can't recall.